Vagrantfile

```diff
@@ -57,13 +57,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
  # This allows developers to view message queues at http://localhost:15672/
  config.vm.network "forwarded_port", guest: 15672, host: 15672
 
- # This is an optional plugin that, if installed, updates the host's /etc/hosts
+ # This is an plugin that, updates the host's /etc/hosts
```

Comment:
This is **a** plugin :-)

Reply:
i pronounce plugin, as `eeep-loog-in` so an fits here.

Reply:
jk. changing it now

Reply:
fixed and force-pushed